### PR TITLE
Add CircleCI build

### PR DESCRIPTION
**Pull request by lbalmaceda**
### Changes

- Revert previous PR that was adding a composer.lock file.
- Add CircleCI build. This should ideally be using both PHP 5.5 and 7.x. But I see no test suite added so I'll just use 7.x to generate the `composer.lock` file required by snyk.

- [ ] This change has been tested on the latest version Laravel

### Checklist

- [x] I have read the [Auth0 general contribution guidelines](https://github.com/auth0/open-source-template/blob/master/GENERAL-CONTRIBUTING.md)

- [x] I have read the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md)

### CODE REVIEW

**Comment by joshcanhelp** [comment]
@lbalmaceda - Can you remove `"minimum-stability": "dev"` from composer.json? That's pulling down directly from GitHub and, for some reason, using an older version of Symfony, which is causing Snyk to fail:

https://getcomposer.org/doc/04-schema.md#minimum-stability

**Comment by lbalmaceda** [comment]
Done :)

**Comment by lbalmaceda** [comment]
Done :)

**Comment by lbalmaceda** [comment]
Done :)

**Comment by lbalmaceda** [comment]
Done :)

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          paths:
+            - vendor
+
+jobs: 
+  php_7: 
+    docker: 
+      - image: circleci/php:7.1
+    steps:
+      - prepare:
+          symfony_version: 3.3.*
```
Comment (joshcanhelp) [inline .circleci/config.yml]: This might be the problem ... I'm guessing this came from the previous CI config? Not sure why this is being frozen at 3.

**Pull request merged**